apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts

```diff
@@ -182,36 +207,66 @@ export class BulkScriptAction extends BaseScriptAction {
           ' operating system and number of CPU cores.'
       });
     }
-    this._toFlag = this.defineStringListParameter({
+    this._toProject = this.defineStringListParameter({
       parameterLongName: '--to',
       parameterShortName: '-t',
+      argumentName: 'PROJECT2',
+      description:
+        'Restricts command execution to the specified project and all dependencies of the specified project. ' +
+        '"." can be used as shorthand to specify the project in the current working directory. ' +
+        'Specifying "--to" or "--to-except" multiple times will take the union of matching projects. ' +
+        'Specifying "--from" as well will further restrict execution to the intersection of both filters.',
+      completions: this._getProjectNames.bind(this)
+    });
```

Comment:
> And you want to rebuild only B and C, there is no way to ask for that to happen.

BTW we could instead solve that with an explicit parameter like `--include` :

```shell
# Build B and C only
$ rush build --include B --include C
```

You could also imagine an `--exclude` parameter:
```shell
# Build A and all its dependents and their dependencies (PR 2349 semantics), but skip B
$ rush build --from A --exclude B
```

Reply:
Supporting the described "build only X and Y" scenario has been postponed to a later change that will focus on the concept of a "project set" or "working set" or "tag" or whatever term we actually decide to use.